src/client/package.json

```diff
@@ -6,5 +6,9 @@
     "gulp": "^3.8.10",
     "gulp-watch": "^3.0.0"
   },
+  "scripts": {
+    "build": "gulp",
```

Comment:
It's demonstrative only - the fe skeleton always replaces this code, or another stack - this is just an opinionated demo of the structure.